Source/Core/VideoBackends/Vulkan/StreamBuffer.cpp

```diff
@@ -131,7 +131,7 @@ bool StreamBuffer::AllocateBuffer()
 
 bool StreamBuffer::ReserveMemory(u32 num_bytes, u32 alignment)
 {
-  const u32 required_bytes = num_bytes + alignment;
+  const u32 required_bytes = Common::AlignUp(num_bytes, alignment);
```

Comment:
Are you sure this is correct? The alignment padding here isn't dependent on the new content size, it is the maximum amount of padding which can be added to the current offset to align it.